sos/report/plugins/fail2ban.py

```diff
@@ -0,0 +1,40 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, DebianPlugin, UbuntuPlugin
+
+
+class Fail2Ban(Plugin, DebianPlugin, UbuntuPlugin):
+
+    short_desc = "Fail2Ban daemon"
+
+    plugin_name = "fail2ban"
+    profiles = ('security',)
+    packages = ('fail2ban',)
+
+    common_dir = '/etc/fail2ban'
+
+    def setup(self):
+
+        self.add_service_status('fail2ban')
+
+        self.add_copy_spec([
+            f'{self.common_dir}/fail2ban.*',
+            f'{self.common_dir}/jail.*',
+            f'{self.common_dir}/filter.d',
+            f'{self.common_dir}/action.d',
+            f'{self.common_dir}/jail.d',
+        ])
+
+        self.add_cmd_output([
+            'fail2ban-client status',
+            'fail2ban-client banned',
+        ])
+
+# vim: et ts=4 sw=4
+
```

Comment:
The `flake8` linting tests are failing on this trailing newline.

Reply:
Removed.